xbmc/guilib/guiinfo/SystemGUIInfo.cpp

```diff
@@ -425,7 +425,8 @@ bool CSystemGUIInfo::GetBool(bool& value, const CGUIListItem *gitem, int context
       value = false;
       return true;
     case SYSTEM_ETHERNET_LINK_ACTIVE:
-      // wtf: not implementated - always returns true?!
+      // not implementated - always returning true is bad practice though
```

Comment:
Let's review the review! s/not implementated/not implemented/

I'm cooking at the same time.

Reply:
Thanks very much for your valuable comment. I wanted to honor the original author, cause he might have a specialy feeling when using "implemented" in a wrong form.

Do you think it is okay to change it?

Reply:
I'm the original author of that bad phrase and I do really sorry for using hate speech and even wrong spelling. Yes, both the hate speech and the typo should be fixed asap. Thanks for pointing this out.